github/transport.go

```diff
@@ -64,8 +64,8 @@ func (rlt *rateLimitTransport) RoundTrip(req *http.Request) (*http.Response, err
 	// If you're making a large number of POST, PATCH, PUT, or DELETE requests
 	// for a single user or client ID, wait at least one second between each request.
 	if rlt.delayNextRequest {
-		log.Printf("[DEBUG] Sleeping %s between write operations", writeDelay)
-		time.Sleep(writeDelay)
+		log.Printf("[DEBUG] Sleeping %s between write operations", rlt.writeDelay)
```

Comment:
```suggestion
		log.Printf("[DEBUG] Sleeping %d between write operations", rlt.writeDelay)
```

Do we need to specify the integer format?

Reply:
I find the string formatting more readable, although I think this is a somewhat subjective opinion. Here's a quick example:

```go
package main

import (
        "log"
        "time"
)

func main() {
        writeDelay := time.Duration(50) * time.Millisecond
        log.Printf("Sleeping %s string seconds between write operations", writeDelay)
        log.Printf("Sleeping %d digit seconds between write operations", writeDelay)
}
```

This program produces the following output: 

![image](https://user-images.githubusercontent.com/9327688/135211722-1cd2f3ea-e495-43f2-8b99-54f6e597711c.png)

I think for this type of unstructured log output, it's nice to have the human-readable suffix attached rather than default to the Go nanosecond unit, which is hard to parse at a glance. What do you think?

Reply:
Agreed, nice touch.